src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -82,6 +82,33 @@ echo "Running '{{ PackageInstallCommand }}'..."
 echo
 {{ PackageInstallCommand }}
 
+{{ if OryxAppInsightInjectCommand | IsNotBlank }}
+echo
+echo "Preparing Envrionment to set up application insight..."
+echo "Running '{{ OryxAppInsightInjectCommand }}'..."
+echo
+{{ OryxAppInsightInjectCommand }}
+echo
+echo "Creating application insight loader file to '$DESTINATION_DIR'"
+echo "// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
```

Comment:
Maybe add another single-line comment like "Added by Oryx"? just to make things clearer